src/meta/meta_http_service.h

```diff
@@ -89,11 +112,17 @@ class meta_http_service : public http_service
     void query_duplication_handler(const http_request &req, http_response &resp);
     void start_bulk_load_handler(const http_request &req, http_response &resp);
     void query_bulk_load_handler(const http_request &req, http_response &resp);
+    void start_compaction_handler(const http_request &req, http_response &resp);
 
 private:
     // set redirect location if current server is not primary
     bool redirect_if_not_primary(const http_request &req, http_response &resp);
 
+    void update_app_env_handler(const std::string &app_name,
```

Comment:
```suggestion
    void update_app_env(const std::string &app_name,
```

the "_handler" suffix is for http handler. this function is not handler.